plugins/grab.py

```diff
@@ -103,13 +100,13 @@ def grab(text, nick, chan, db, conn):
             # check to see if the quote has been added
             if check_grabs(name.lower(), msg, chan):
                 return "I already have that quote from {} in the database".format(text)
-                break
```

Comment:
The `return` breaks out of the function at that point, the `break` is completely unreachable